core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -236,7 +236,7 @@ func (f *FwdMgr) runLoop() {
 	defer f.wg.Done()
 	tick := time.After(0)
 
-	for ; ; tick = time.After(utils.WithJitter(time.Duration(time.Minute))) {
+	for ; ; tick = time.After(utils.WithJitter(time.Minute)) {
```

Comment:
Unrelated to the scope of this PR - is there a more idiomatic way to write this?

Reply:
It does seem odd. Moving the assignment to the body isn't straightforward though (extra bool might be most elegant?).

I might actually prefer moving the init here:
```suggestion
	for tick := time.After(0); ; tick = time.After(utils.WithJitter(time.Minute)) {
```

Or, since this is a common pattern that we use instead of a plain `time.Ticker`, we could just make a custom type of Ticker which sends the first tick without delay. The jitter can become an implementation detail this way too:
```suggestion
	ticker := services.NewTicker(time.Minute)
	defer ticker.Stop()
	for {
```

Reply:
Here is a quick hack: https://github.com/smartcontractkit/chainlink-common/pull/311

Reply:
Nice!